drivers/hud_twenty_twenty_two_to_twenty_twenty_four/app/models/hud_twenty_twenty_two_to_twenty_twenty_four/kiba/csv_base.rb

```diff
@@ -29,7 +34,7 @@ def self.up(source_name, destination_name, encoding, header_converter, reference
         Kiba::Common::Destinations::CSV,
         {
           filename: destination_name,
-          headers: target_class.hmis_configuration(version: '2024').keys.map(&:to_s),
+          headers: (target_class.hmis_configuration(version: '2024').keys.map(&:to_s) + preserve_headers).uniq,
```

Comment:
That makes sense, thanks @awisema - I modified the CsvTransformer, it now has a `destination_headers` method that we override in a subclass for AcHmis which is passed into the csv classes. I also converted the csv methods to use named params since positional params were getting kind of long with this addition.